notebooks/week1/intermediate_notebook.py

```diff
@@ -24,6 +24,7 @@
 
 parent_dir = os.path.dirname(os.getcwd())
 sys.path.append(parent_dir)
```

Comment:
let's not use sys path

Reply:
Oh, this is the intermediate notebook, so it’s only half good, haha. I’ve removed this from the prettier notebook.